parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -28,9 +28,15 @@ done
 sed -i "/#HELPERSEOF/d" {{GetCSEHelpersScriptFilepath}}
 source {{GetCSEHelpersScriptFilepath}}
 
+wait_for_file 3600 1 {{GetCSEHelpersScriptDistroFilepath}} || exit $ERR_FILE_WATCH_TIMEOUT
+source {{GetCSEHelpersScriptDistroFilepath}}
```

Comment:
maybe also use the HELPERSEOF to indicate file existence?
sed -i "/#HELPERSEOF/d" {{GetCSEHelpersScriptFilepath}}

Reply:
I think we can just rely on the wait_for_file macro here since its defined by the common cse_helpers.sh that's been sourced above. I've updated cse_helpers_mariner and cse_helpers_ubuntu to end with #EOF instead of #HELPERSEOF so it can use the macro correctly.